app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -154,8 +126,77 @@ class _ChatBubble extends ConsumerWidget {
+                child: message.repliedMessage != null
+                    ? Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: <Widget>[
+                          Container(
+                            decoration: BoxDecoration(
+                              color: Theme.of(context)
+                                  .colorScheme
+                                  .neutral
+                                  .withOpacity(0.3),
+                              borderRadius: BorderRadius.circular(22),
+                            ),
+                            child: Column(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: [
+                                Padding(
+                                  padding: const EdgeInsets.only(
+                                    left: 10,
+                                    top: 15,
+                                  ),
+                                  child: Row(
+                                    children: [
+                                      ActerAvatar(
+                                        uniqueId:
+                                            message.repliedMessage!.author.id,
+                                        mode: DisplayMode.User,
+                                        displayName: message
+                                            .repliedMessage!.author.firstName,
+                                        avatar: ref
+                                            .watch(
+                                              chatRoomProvider.notifier,
+                                            )
+                                            .getUserProfile(
+                                              message.repliedMessage!.author.id,
+                                            )
+                                            ?.getAvatarImage(),
+                                        size: 12,
+                                      ),
+                                      const SizedBox(width: 5),
+                                      Text(
+                                        '${message.repliedMessage?.author.firstName}',
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .bodySmall!
+                                            .copyWith(
+                                              color: Theme.of(context)
+                                                  .colorScheme
+                                                  .tertiary,
+                                            ),
+                                      ),
+                                    ],
+                                  ),
+                                ),
+                                _OriginalMessageBuilder(message: message),
```

Comment:
I am wondering though, this can be arbitrary large messages ... should we cap this somehow? Only show the top 100px or something... but maybe we just wait until we see that as a problem being reported...